src/components/ProgressIndicator.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const ProgressIndicator = () => (
+	<div className= "progressIndicator"></div>
+);
+
```

Comment:
When you have a stateless component, you need to give it a display name after the component. Add `ProgressIndicator.displayName = 'ProgressIndicator'` here and your tests will pass ;)